### ISSUE TITLE
First timer

### ISSUE THREAD

**Issue by arctracerexequias** [arctracerexequias/sorts#1]
Ow yeah! First time here in GitHub. I'll try to contribute something to GitHub. XD

**Comment by arkeidolon** [comment]
Magpush ka na. Tapos magcocontribute din ako.

**Comment by arctracerexequias** [comment]
yeah. dito ko ilalagay ung sa PPL namin at ung sa Tech Doc namin. :smile:
